prestoadmin/server.py

```diff
@@ -125,6 +127,15 @@ def uninstall():
         print('Package uninstalled successfully on: ' + env.host)
 
 
+@task
+@requires_topology
+def upgrade(local_package_path, local_config_dir):
+    configure_cmds.gather_directory(local_config_dir)
+    uninstall()
```

Comment:
could you change this to do a package upgrade (i.e. ultimately call out to rpm -u).  This would allow us to be smarter about the things we erase or don't erase depending on if it's an upgrade.  Right now we don't try to do anything smart, but it would probably be good to do in the future.